odl/solvers/iterative/statistical.py

```diff
@@ -121,6 +138,131 @@ def mlem(op, x, data, niter=1, noise='poisson', callback=None):
         raise RuntimeError('unknown noise model')
 
 
+def osmlem(op, x, data, niter=1, noise='poisson', callback=None, **kwargs):
```

Comment:
Page 122 in [Mathematical Methods in Image Reconstruction, by F. Natterer and F. Wübelling, SIAM, 2001](http://epubs.siam.org/doi/book/10.1137/1.9780898718324) is in my opinion the best reference for OS-MLEM. Short and precise, while pointing out the difficulties in assessing convergence properties.